content/news/intrapixel-detector-SPIE.md

```diff
@@ -0,0 +1,20 @@
+Title: Intrapixel Detector Responsivity Measured on Kepler Flight Spare
+Date: 2018-07-23 09:00
+Author: Michael Gully-Santiago
+
+
+
+Researchers have recently characterized the Intra-Pixel Response Function (IPRF) of a Kepler [flight spare](https://en.wikipedia.org/wiki/Flight_spare) CCD.  Recent [Center for Imaging Science](https://www.cis.rit.edu/) PhD recipient Dmitry Vorobiev presented the study at the SPIE meeting on Astronomical Telescopes and Instrumentation in Austin, Texas in June 2018.  The [findings are posted on ArXiV](https://arxiv.org/abs/1806.07430) and will appear in the upcoming Proceedings of the SPIE.
+
+The Intra-Pixel Response Function is the mapping of detector sensitivity within a pixel boundary.  The IPRF imbues small (hundreds of parts-per-million) flux deficits and excesses as spacecraft-induced motion drags celestial photo-centers within a pixel.  These conspicuous instrumental flux changes dominate the variance of K2 aperture photometry.  [Several detrending algorithms](http://lightkurve.keplerscience.org/other_software.html#detrending-analysis) account for these sawtooth-like artifacts in the flux time series.  The lab-based measurement of the IPRF will enable forward-modeling approaches such as Point Spread Function (PSF) photometry, needed for precision photometry in high stellar density fields.  Alternatively, quantifying this sub-pixel flat field can enhance the fidelity of [synthetic K2 photometry](https://nksaunders.github.io/files/Example.html), which can inform the reliability of data-driven heuristic methods.  
```

Comment:
Instead of `hundreds of parts-per-million`, do you want to express the flux changes in relative units i.e. percent?

Reply:
Ok done.  I deleted the parts-per-million bit entirely because there are several assumptions and qualifiers built into that estimate, so better to simply leave out the estimate altogether.